src/coreclr/gc/gc.cpp

```diff
@@ -25402,17 +25402,21 @@ void gc_heap::calculate_new_heap_count ()
     // If there was a blocking gen2 GC, the overhead would be very large and most likely we would not pick it. So we
     // rely on the gen2 sample's overhead calculated above.
     float throughput_cost_percents[dynamic_heap_count_data_t::sample_size];
-    for (int i = 0; i < dynamic_heap_count_data_t::sample_size; i++)
-    {
-        dynamic_heap_count_data_t::sample& sample = dynamic_heap_count_data.samples[i];
-        assert (sample.elapsed_between_gcs > 0);
-        throughput_cost_percents[i] = (sample.elapsed_between_gcs ? (((float)sample.msl_wait_time / n_heaps + sample.gc_pause_time) * 100.0f / (float)sample.elapsed_between_gcs) : 0.0f);
-        assert (throughput_cost_percents[i] >= 0.0);
-        if (throughput_cost_percents[i] > 100.0)
-            throughput_cost_percents[i] = 100.0;
-        dprintf (6666, ("sample %d in GC#%Id msl %I64d / %d + pause %I64d / elapsed %I64d = tcp: %.3f, surv %zd, gc speed %zd/ms", i,
-            sample.gc_index, sample.msl_wait_time, n_heaps, sample.gc_pause_time, sample.elapsed_between_gcs, throughput_cost_percents[i],
-            sample.gc_survived_size, (sample.gc_pause_time ? (sample.gc_survived_size * 1000 / sample.gc_pause_time) : 0)));
+
+    if (process_eph_samples_p)
```

Comment:
so is the `process_eph_samples_p` check the only change. weird its showing up as a larger change.

Reply:
Correct - I think the fact we indented the for-loop, we see those as changed lines.

Reply:
Btw, switching to hide the whitespace should highlight _just_ the added changes:

![image](https://github.com/user-attachments/assets/72fd62ee-a4a3-4e99-bf39-797d932146b8)